dbschema/partner.esdl

```diff
@@ -0,0 +1,38 @@
+module default {
+  type Partner extending Resource, Project::ContextAware, Mixin::Pinnable, Mixin::Taggable {
```

Comment:
I think it would be better if this was `Named`. It's really annoying to not have this. I think it will most immediately affect is search. You can see all the references to "PartnerByOrg" in https://github.com/SeedCompany/cord-api-v3/blob/9f47163ab06438265a7e8dff549522ab74bcc596/src/components/search/search.service.ts where we work around this.

We could just have a trigger on Organization to update this on the related Partner.

Speaking of which I don't even see organization linked here....on nvm it's buried in the other links.

Reply:
Yeah, I was going to put it on there, but I was following the current model which doesn't have `name` on `Partner`.  It does seem odd not to have it.

The trigger idea is nice.

Reply:
I think I just didn't want to bother with Neo4j. Here it's much easier.